docs/csp.md

```diff
@@ -41,45 +50,55 @@ app.use(
 When loading the initial index page, the `Content-Security-Policy` header should now contain the nonce:
 
 ```
-style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3'
+default-src 'self'; style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3';
 ```
 
-2. In webpack config:
+1. Now we want to include this value in our page so that JSS can pick it up at runtime.
+
+You can use any templating engine, or if you have SSR that should work too. For this example we'll use the [Nunjucks](https://github.com/mozilla/nunjucks) template engine.
 
-Set the `global.__webpack_nonce__` variable to the same nonce value:
+1. First create the template html file.
+
+```html
+<head>
+  <meta id="jss-nonce" data-value="{{ noncePlaceholder }}">
```

Comment:
Maybe a more universal name is appropriate? Also `noncePlaceholder` should be `styleNoncePlaceholder` or `styleNonce`.

Reply:
Not sure what you mean by a "more universal name" ?

Reply:
Thats what I ment by `<meta property="nonce" content="asdfasdfaasdf" />` nonce can be used for anything else on the page that requires nonce.

Reply:
For instance, in my most recent project, I have a  nonce for js and one for css. They should be different. OAuth and APIs have the concept of nonces as well. By universal I mean someone could just copy this code and it would work. It’s possible someone already has a meta tag “nonce” for something else. Simply being specific is all.

Reply:
Why exactly should they be different? I thought nonce is nonce, no matter for what it is used?

Reply:
You _can_ use different nonces for script vs style, but you don't have to. But just "nonce" on it's own is probably too generic, nonces can be used for anything, not limited to CSP context.

Reply:
ok, what about "csp-nonce" as a name, it is still for more than JSS, but seems now specific enought to identify properly the nonce value, no?

Reply:
is there any reason one would use different nonces for js and for css?

Reply:
Where I am heading with this, in case one uses more than one library on the page, for e.g. JSS and styled-components, they could share the same nonce token.

Reply:
I like your idea. I think `csp-nonce` is probably the best variable to use across the board.

Reply:
so then `<meta property="csp-nonce" content="asdfasdfaasdf" />`

Reply:
Updated